daemon/src.cmake

```diff
@@ -301,7 +301,7 @@ set(QCOMMONLIST
     ${ENGINE_DIR}/sys/con_common.cpp
 )
 
-if (NOT APPLE)
+if (NOT APPLE AND (USE_CURSES OR WIN32))
```

Comment:
Instead of so many checks, we could change the default value of USE_CURSES based on the OS and then just use that. (this would affect the other NOT APPLE above).